src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,178 @@
+import spacy
```

Comment:
Please add the license header and `#!/usr/bin/env python3` to top and add the execute permission to file with `chmod +x`

Reply:
Please add license headers to the file and do a `chmod +x copyrightDeactivationClutterRemovalScript.py` on it before committing.